src/lib.rs

```diff
@@ -37,11 +37,6 @@ use core::convert;
 use core::fmt;
 use core::ops;
 
-#[cfg(feature = "std")]
-use core::io;
-
-/// A digest.
-#[derive(Clone, Copy, Eq, Hash, PartialEq)]
```

Comment:
Please keep the docstring and the derive.

Reply:
Done